index.js

```diff
@@ -0,0 +1,63 @@
+/** @format */
+
+const bookList = document.querySelector('.booklist');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const addBtn = document.querySelector('#addBtn');
+let userBooks = JSON.parse(localStorage.getItem('userBooks')) || [];
+
+// setItems
+function addBook() {
+  if (title.value === '' && author.value === '') {
+    alert('Please fill up all fields');
+  } else {
+    userBooks.push({
+      id: Math.floor(Math.random() * 1000),
+      bookTitle: title.value,
+      bookAuthor: author.value,
+    });
+    localStorage.setItem('userBooks', JSON.stringify(userBooks));
+  }
+}
+
+// Displaying
+function showList() {
+  const books = JSON.parse(localStorage.getItem('userBooks'));
+  let content = '';
+  books.forEach((book) => {
+    content += `
+    <div>
+          <p class="title">${book.bookTitle}</p>
+          <p class="author">${book.bookAuthor}</p><br>
+          <button type="submit" class="removeBtn" id="${book.id}">Remove</button>
+          <br>
+    </div>
+    `;
+  });
+  bookList.innerHTML = content;
+}
+
+// getItems
+window.addEventListener('load', () => {
+  showList();
+});
+
+// remove
+document.addEventListener('click', (e) => {
+  if (e.target.classList.contains('removeBtn')) {
+    userBooks = userBooks.filter((book) => book.id.toString() !== e.target.id);
+
+    localStorage.setItem('userBooks', JSON.stringify(userBooks));
+  }
+  showList();
+});
```

Comment:
- [ ] You're meant to create a function for the `remove` functionality of your app. This is not a standalone function. Its an event listener that listens for a 'click' event on the document object, and when a click event is fired, it checks if the clicked element has a class of 'removeBtn` and then does the filter call.
- Kindly implement the same functionality as a function 👍 

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/235511820-7663b023-9dc6-48a0-a865-445b8422024c.png)


## Suggested solution:
```
function handleRemoveClick(e) {
  if (e.target.classList.contains('removeBtn')) {
    userBooks = userBooks.filter((book) => book.id.toString() !== e.target.id);

    localStorage.setItem('userBooks', JSON.stringify(userBooks));
  }
  showList();
}

document.addEventListener('click', handleRemoveClick);
```